simtbx/diffBragg/tests/tst_diffBragg_diffuse_properties.py

```diff
@@ -11,15 +11,25 @@
 parser.add_argument("--sigma", default=[1,1,1], type=float, nargs=3)
 parser.add_argument("--gamma", default=[100,100,100], type=float, nargs=3)
 parser.add_argument("--idx", type=int, default=0, choices=[0,1,2], help="diffuse parameter index (0,1,2 ->a,b,c)")
+parser.add_argument("--stencil", type=int, default=0, help="sets the stencil property in diffBragg (default is 0)")
+parser.add_argument("--laue", action="store_true", help="sets the laue group number for the spacegroup")
 parser.add_argument("--grad", choices=['sigma','gamma'], default='gamma')
 
 import pylab as plt
+import os
+
 args = parser.parse_args()
 if args.cuda:
-    import os
     os.environ["DIFFBRAGG_USE_CUDA"] = "1"
 
+if os.environ.get("DIFFBRAGG_USE_CUDA") is None:
```

Comment:
@dermen We can remove this test now, correct?

Reply:
I believe so, though I havent tested myself

Reply:
OK. I think the CPU code should be ready to use now, so let's just use it here...

I can do the edits tomorrow morning (for me) if you don't get around to it.

Reply:
Updated.